scripts/upgrade

```diff
@@ -78,10 +78,16 @@ if [ "$upgrade_type" == "UPGRADE_APP" ]
 then
 	ynh_script_progression --message="Upgrading source files..." --weight=6
 
+	ynh_secure_remove --file="$final_path"
```

Comment:
you should backup and restore `/var/www/moodle/admin/cli/../../config.php` during source upgrade ^^